libs/base/ui/graphical/rating/src/styles/base.styles.ts

```diff
@@ -50,7 +50,7 @@ export const ratingBaseStyles = css`
 
   fieldset slot,
   fieldset ::slotted(*) {
-    color: var(--oryx-rating-color-active, var(--oryx-color-secondary-300));
+    color: var(--oryx-rating-color-active, var(--oryx-color-secondary-9));
```

Comment:
primaryBaseColor

Reply:
nope, this is secondary. This PR is only about primary.